staff/felipe-fernandez/blog-layout/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Blog</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+        <header>
+            <h1>
+               My Blog
+            </h1>
+        </header>  
+
+        <nav>
+            <ul>
+                <li>Articles</li>
+                <li>About Me</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+      
```

Comment:
It would be more semantic if you included all your main content in a <main> element. Your body structure then would be: <header> <main> <footer>. You can include your navigation menu in the header and put the remaining sections inside <main>.

Reply:
Fixed! Thanks for the correction.